### PR TITLE
Manage client connection and style updates

### PR DESCRIPTION
**Pull request by ajself**
- Allow the applications to disconnect the connect. This allows apps with users who sign in/out or have multiple accounts the ability to discontinue receiving messages for an unrelated account.

- Add more style flexibility for default cells

### CODE REVIEW

File: Sources/Core/Client/Client.swift [hunk 1]
```diff
@@ -121,7 +121,7 @@ public final class Client {
         }
     }
     
-    func reset() {
+    public func reset() {
```
Comment (buh) [inline Sources/Core/Client/Client.swift]: Looks good.
Can you add a doc for the method and point developers to use `Client.set(user:, token:)` to restore the connection.

Comment (buh) [reply Sources/Core/Client/Client.swift]: And now better to rename it to `disconnect()`.

Comment (ajself) [reply Sources/Core/Client/Client.swift]: Agreed and good idea. Will update.

**Comment by ajself** [comment]
@buh Updated

**Pull request merged**